### DISCUSSION TITLE
已经配置了key了，还是请求超时

### DISCUSSION THREAD

**Discussion by Lcicode** [MuiseDestiny/zotero-gpt#146] [General]
![image](https://github.com/MuiseDestiny/zotero-gpt/assets/131152247/c0c794c0-a7c5-48fb-b679-affb6f7d0e49)
![image](https://github.com/MuiseDestiny/zotero-gpt/assets/131152247/3dd11ce2-d9f5-4a87-8aa1-43df93bf9d3d)

**Comment by MuiseDestiny** [comment]
网络不畅。开启代理，切换代理。
